Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentStart/ReaderMode.js

```diff
@@ -79,6 +79,8 @@ function checkReadability() {
       readabilityResult.title = escapeHTML(readabilityResult.title);
       // Sanitize the credits to prevent a malicious page from inserting HTML in the `<credits>`.
       readabilityResult.credits = escapeHTML(readabilityResult.credits);
```

Comment:
I just noticed that `credits` is always undefined. It seems that it's [not returned](https://github.com/mozilla/readability/blob/main/README.md?plain=1#L41-L53) by the API ( maybe it legacy field ? )